docs/staking-modules/csm/guides/addresses.md

```diff
@@ -98,4 +100,33 @@ These changes grant `managerAddress` the ultimate control over the Node Operator
 
 :::info
 If the `extendedManagerPermissions` option is used, `managerAddress` should be set to a secure address ("cold" wallet or multisig)
-:::
\ No newline at end of file
+:::
+
+## Examples
+
+### Default
+
+By default, the Node Operator's creator is set as a `managerAddress` and `rewardAddress`.
+
+![addresses-2](../../../../static/img/csm/addresses-2.png)
+
+It is recommended that addresses be re-assigned according to one of the optimal configurations below.
+
+### Optimal for solo operator
+
+It is considered optimal to set `managerAddress` to a "Hot" or easily accessible address and `rewardAddress` to a secure "Cold" wallet or even a MultiSig if you are a solo operator.
+
+![addresses-3](../../../../static/img/csm/addresses-3.png)
+
+### Optimal for DVT operator
+
+Several optimal options exist for the DVT operators.
+
+The first is to set `managerAddress` to a "Hot" or just easily accessible address controlled by the cluster coordinator and `rewardAddress` to a cluster MultiSig.
+
+![addresses-4](../../../../static/img/csm/addresses-4.png)
+
+
+The second option is to set `managerAddress` to a cluster MultiSig, `rewardAddress` to a splitter contract, and enable `extendedManagerPermissions`.
```

Comment:
"to a splitter contract," => "to a splitter contract (e.g. to benefit from automated rewards splitting between cluster participants),"